Python/pylifecycle.c

```diff
@@ -1157,6 +1157,28 @@ init_interp_main(PyThreadState *tstate)
         return status;
     }
 
+#ifdef Py_DEBUG
+    if (config->run_presite) {
```

Comment:
Can you add a function for this code to keep init_interp_main() short?

Reply:
Added in the upcoming push.